spec/requests/providers/online_bankings_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+RSpec.describe 'does client use online banking requests', type: :request do
+  let(:applicant) { create :applicant }
+  let(:application) { create :legal_aid_application, applicant: applicant }
+  let(:application_id) { application.id }
+
+  describe 'GET /providers/applications/:legal_aid_application_id/applicant' do
+    let(:get_request) { get "/providers/applications/#{application_id}/does-client-use-online-banking" }
+
+    it 'returns http success' do
+      get_request
+      expect(response).to have_http_status(:ok)
+    end
+
+    context 'when the application does not exist' do
+      let(:application_id) { SecureRandom.uuid }
+
+      it 'redirects the user to the applications page with an error message' do
+        get_request
+        expect(response).to redirect_to(providers_legal_aid_applications_path)
+      end
+    end
+
+    it 'displays the correct page' do
+      get_request
+      expect(unescaped_response_body).to include('Does your client use online banking?')
+    end
```

Comment:
A pedantic point perhaps, but I think `it` within `context`s should be clearly separated from those without context. Personally I like the pattern:
```ruby
describe 'the main container describe the scope of the spec' do
  it 'specs that describe default or no parameters entered behaviour` do
  end

  context 'behaviours that deviate from default/no-parameter' do
  end
end
```
In other words put all the `it` blocks that are not modified by contexts at the start of the description block, and start the context blocks after all of the not-context `it` blocks.